### DISCUSSION TITLE
Stuck on "Fetching Channel(s)..."

### DISCUSSION THREAD

**Discussion by OliverH1** [Tyrrrz/DiscordChatExporter#616] [General]
Hey guys,

So whenever I try to run the CLI version of DiscordChatExporter on my machine with Windows 10 x64, it just keeps stuck at "Fetching Channel(s)...".

The GUI version on the other hand, shows a notification on the bottom saying "Failed to perform application update".

![failed to update](https://user-images.githubusercontent.com/86267774/122832156-7a361400-d2da-11eb-9d70-c411a52eddc0.png)

And then, a while after I enter my token and press enter, it gives me this message:

![error](https://user-images.githubusercontent.com/86267774/122832232-9639b580-d2da-11eb-9e0c-65ebc3164117.png)

Now, I have tried a couple of things to fix this, such as:
- Trying older versions of DiscordChatExporter
- Uninstalling and Installing different versions of .NET Core v3.1 Runtime
- Disabling IPV6
- Switching back and forth between ISP and Google DNS
- Disabling Antivirus
- Disabling Firewall

The thing is, DiscordChatExporter used to work on this machine just fine, but it stopped working out of the sudden like 6 months ago or so.

I'm now running both versions of DiscordChatExporter (GUI & CLI) on a Windows 7 x64 virtual machine and it's working just fine.
Just like my main machine, this VM is running the DiscordChatExporter 2.28 and .NET Desktop Runtime 3.1.16 as you can check on the following picture (output from dotnet --info):

![dotnet info](https://user-images.githubusercontent.com/86267774/122832045-4e1a9300-d2da-11eb-8100-07a56d39323c.png)

Could you help me identify where the problem is?

**Comment by Tyrrrz** [comment]
Looks like a connectivity error, but not sure what caused it. I tested DiscordChatExporter on clean Windows 10 x64 install a few weeks ago and it worked out of the box. I don't know what else to suggest besides reinstalling/resetting Windows.

**Reply by OliverH1** [reply]
Thank you for taking the time to reply to my query.


After some more testing, I found out the issue has to due with the fact that .NET Core 3.1 does not currently support TLS 1.3, thus it needs to be disabled.

In this archive, there are 2 registry files that will help you enable/disable TLS 1.3 as you may see fit.
[Enable & Disable TLS 1.3.zip](https://github.com/Tyrrrz/DiscordChatExporter/files/6695592/Enable.Disable.TLS.1.3.zip)

Hopefully this helps someone else who experiences this same problem.

**Reply by Tyrrrz** [reply]
Ooh, that's interesting. Thanks for letting me know. In the future, this should hopefully be fixed by #521.
